include/javabind/enum.hpp

```diff
@@ -20,13 +20,32 @@
 
 namespace javabind
 {
+    struct EnumValue
+    {
+        jobject object;
```

Comment:
I know the library doesn't currently support load/unload cycles but we might want to think about pairing `initialize` with a `terminate` that deallocates all global references passed to `EnumValues`. We might also consider wrapping the following code into a static function:

```
values.emplace(name, EnumValue { env->NewGlobalRef(value), ordinal });
```

If we do, the ownership of the global reference might become more clear. For example, we could pass the Java environment, allocate the global reference in `EnumValues`, and then release the reference also in `EnumValues`.

This way, we will have an easier task when we implement `JNI_OnUnload`.